### ISSUE TITLE
sunshine.conf written without line break causing config not to take

### ISSUE THREAD

**Issue by alanmccallum** [Nonary/ResolutionAutomation#8]
I was having some issues getting this working.

Attempted to solve this by using the precommand_version branch.

When I still couldn't get it working I ran the ResolutionMatcher.ps1 command manually. Suddenly my screen resized!
I did some more poking around in the code, figured out how the precommand is supposed triggered by sunshine, and investigated the config page, then the config file.

Sunshine.conf looked like this before my fix:
```
min_log_level = 1global_prep_cmd = [{"do":"powershell.exe -executionpolicy bypass -file \"C:\\Users\\{USER}\\Desktop\\ResolutionAutomation\\ResolutionMatcher.ps1\"","elevated":"false","undo":"powershell.exe -executionpolicy bypass -file \"C:\\Users\\{USER}\\Desktop\\ResolutionAutomation\\ResolutionMatcher-Functions.ps1\" True"}]
```

Changing it to this and restarting sunshine worked:
```
min_log_level = 1
global_prep_cmd = [{"do":"powershell.exe -executionpolicy bypass -file \"C:\\Users\\{USER}\\Desktop\\ResolutionAutomation\\ResolutionMatcher.ps1\"","elevated":"false","undo":"powershell.exe -executionpolicy bypass -file \"C:\\Users\\{USER}\\Desktop\\ResolutionAutomation\\ResolutionMatcher-Functions.ps1\" True"}]
```

**Comment by alanmccallum** [comment]
I think a new line character needs to be added to https://github.com/Nonary/ResolutionAutomation/blob/c43661caad11e1273a49f079018facdaab00803e/Install_as_Precommand.ps1#L135

**Comment by Nonary** [comment]
Fixed with latest commit fcd5f48dc6c758a0523c8e45c4cb471aa4a0e628
